test/functional/feature_dbcrash.py

```diff
@@ -218,7 +218,7 @@ def run_test(self):
 
         # Start by creating a lot of utxos on node3
         initial_height = self.nodes[3].getblockcount()
-        utxo_list = create_confirmed_utxos(self.nodes[3].getnetworkinfo()['relayfee'], self.nodes[3], 5000, self.signblockprivkey)
+        utxo_list = create_confirmed_utxos(self.nodes[3].getnetworkinfo()['relayfee'], self.nodes[3], 5000, self.signblockprivkey_wif)
```

Comment:
Why this change needs?

Reply:
This change should have been done earlier when all 'generate' rpcs were changed.  
 They are some long running scripts listed as EXTENDED_SCRIPTS. These are not run by default by the CI. It has to be enabled in configure So I did not notice them until now.
I wanted this change to be a separate PR after checking the running time. But I had to add them to this PR as this test needs it to execute correctly.

Reply:
Create another PR as above.

Reply:
This PR is now clean i.e. only hash serialized bug fix and tis tests are present. Test case 'feature_dbcrash.py' is not executed in the CI now.
I am in the process of fixing the extended functional tests. This will be in a separate PR. After that one is merged all python tests will run in the CI every time.